### DISCUSSION TITLE
Generate report timeout on client while server shows success-200

### DISCUSSION THREAD

**Discussion by dudiavraham3** [allure-framework/allure2#2315] [Questions & Support]
when running
GET /allure-docker-service/generate-report?project_id=sonic-moose-mtvr-moose-01

on this specific project

via python request module (with timeout 120 seconds)
getting timeout exceeded although on the server side it shows response 200 success (after 3 seconds)

more facts:
-on the server log we see "cannot access ../latest/history" when Creating history on the results directory just before the generation
(maybe it is related)
- when running the generate via the swagger it finishes after around 10 seconds

environment:
-allure docker service on k8s with 10 pods
-parameters:
KEEP_HISTORY=1
CHECK_RESULTS_EVERY_SECONDS=NONE

KEEP_HISTORY_LATEST=500
DEV_MODE=1

![client and server call](https://github.com/allure-framework/allure2/assets/10
![error](https://github.com/allure-framework/allure2/assets/101575460/a44d5bca-27c8-4ef3-996b-ff405479ff7b)
1575460/6daf1b58-5a96-497c-bb5f-e72facff1065)

**Comment by baev** [comment]
This is related to 3rd party solution `allure-docker-service`

**Reply by dudiavraham3** [reply]
yes but what can cause ".../projects/{project_id}/reports/latest/history no such file or directory " ?

**Reply by baev** [reply]
I have no idea. Try reading the readme at https://github.com/fescobar/allure-docker-service. Maybe you'll find the solution there.
